File: Python/Single Responsibility Principle/Kullanici.py
class Kullanici():
    def __init__(self):
        self.isim = None
        self.mail = None
        self.adres = None

    def setisim(self, isim):
        self.isim = isim

    def getisim(self):
        return self.isim

    def setmail(self, mail):
        self.mail = mail

    def getmail(self):
        return self.mail

    def adresguncelle(self, adres):
        self.adres = adres

    def getadres(self):
        return str(self.adres)

    def __str__(self):
        return f"{self.getisim()} {self.getmail()} {self.getadres()}"

    def giris_yap(self, mail, sifre):
        print("Giriş yapıldı.")

File: Java/Single Responsibility Principle/Kullanici.java
public class Kullanici{
    private  String isim;
    private  String mail;
    private  Adres adres;

    public Kullanici(String isim, String mail, Adres adres){
        this.isim=isim;
        this.mail=mail;
        this.adres=adres;
    }


    public void girisYap(String isim, String mail){
        System.out.println("Kullanıcı Giriş Yaptı!");
    }
    @Override
    public String toString(){
        return (isim+" "+mail+" "+adres);
    }
   
    public String getIsim(){
        return isim;
    }

    public String getmail(){
        return mail;
    }

    public Adres getadres(){
        return adres;
    }
}